recipes/fakechroot/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+./autogen.sh
+./configure --prefix $PREFIX
+
+make -j $CPU_COUNT
+set +e
```

Comment:
Why?

Reply:
I wanted to run the tests, but it looks like they can only run successfully as the root user.

Let me investigate this further and I'll ping you again. I think I need the "fake root" library to actually run the tests correctly.